src/NRediSearch/Aggregation/AggregationBuilder.cs

```diff
@@ -0,0 +1,157 @@
+﻿// .NET port of https://github.com/RedisLabs/JRediSearch/
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Linq;
+using NRediSearch.Aggregation.Reducers;
+
+namespace NRediSearch.Aggregation
+{
+    public class AggregationBuilder
```

Comment:
if this isn't using inheritance, we should probably mark this `sealed` until there's a good reason to un-seal it